app-libs/stf/src/getter.rs

```diff
@@ -283,6 +301,11 @@ impl ExecuteGetter for PublicGetter {
 				};
 				Some(parentchains_info.encode())
 			},
+			PublicGetter::note_buckets_info => {
+				let first_bucket = Notes::buckets(0);
+				let last_bucket = Notes::buckets(0);
+				Some((first_bucket, last_bucket).encode())
```

Comment:
Can we maybe introduce a type like:

```rust
pub struct NotesBuckets {
   first: BucketInfo,
   last: BucketInfo,
}
```

Tuples are a pain to handle in many programming languages.

Reply:
Notabene, it is already defined as a struct in https://github.com/encointer/encointer-js/pull/120 because the scale codec of tuples is compatible with structs.

Reply:
Note: there was an error in 17.0.0-alpha.0 where first, last wasn't optional. Fixed this in 17.0.0-alpha.1